src/attributes.jl

```diff
@@ -96,6 +96,11 @@ If an `F`-in-`S` constraint with name `name` exists in the model `model`, return
 
 If *any* constraint with name `name` exists in the model `model`, return the corresponding index, otherwise throw a `KeyError`. This version is available for convenience but may incur a performance penalty because it is not type stable.
 
+### Note
+
+It is the user's responsibility to check that `canget` returns `true` prior to calling `get`. 
+Calling `get` when `canget` returns `false` will throw an error.
```

Comment:
This is now a weird statement. Do we need to add this note for `addconstraint!` `modifyconstraint!`, etc (anything that as a corresponding `canXXX`)?
I would phrase it more of a note for implementations. These functions *must* throw errors if the corresponding `canXXX` returned false.

Reply:
Should this just be a general note in the manual then?

Reply:
Yeah, could be in the section on notes for solver wrappers.